src/meta/service/src/ddl_service.rs

```diff
@@ -1172,6 +1172,31 @@ impl DdlService for DdlServiceImpl {
 
         Ok(Response::new(AlterResourceGroupResponse {}))
     }
+
+    async fn alter_database_param(
+        &self,
+        request: Request<AlterDatabaseParamRequest>,
+    ) -> Result<Response<AlterDatabaseParamResponse>, Status> {
+        let req = request.into_inner();
+        let database_id = req.database_id;
+
+        if let Some(config_param) = req.param {
```

Comment:
We can assert that the `req.param` must be `Some`. Usually we do `req.param.unwrap()` here.